database.py

```diff
@@ -19,12 +19,12 @@
 
 class DB():
 
-	def __init__():
+	def __init__(self):
 		self.client = MongoClient('mongodb://localhost:27017/')
-		self.db = client.mentor_tool
+		self.db = self.client.mentor_tool
 		self.user_table = self.db.user_table
 
-	def insertPost(user_data):
+	def insertPost(self, user_data):
```

Comment:
backend service passes JSON object to DB API, so here the input of each API is JSON.